ai/embedding/service.py

```
@@ -0,0 +1,181 @@
+            logger.info(f"Job vector created - General: {general_vector.shape}, Skills: {skills_vector.shape}, Combined: {combined_vector.shape}")
+
+            return VectorResult(
+                general_vector=general_vector.tolist(),
+                skills_vector=skills_vector.tolist(),
+                combined_vector=combined_vector.tolist(),
+                model=model_name,
+                dimension=dimension
+            )
+
+        except Exception as e:
+            logger.error(f"Failed to create job vector: {e}")
```

Ruff (0.13.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)